Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -862,6 +848,20 @@ void CFrame::DoStop()
     if (NetPlayDialog::GetNetPlayClient())
       NetPlayDialog::GetNetPlayClient()->Stop();
 
+    const auto& stm = WII_IPC_HLE_Interface::GetDeviceByName("/dev/stm/eventhook");
```

Comment:
You may want to try factoring out this block into its own named function, if possible.